pkg/exporter/sumologicexporter/sender_test.go

```diff
@@ -60,6 +60,7 @@ func prepareSenderTest(t *testing.T, cb []func(w http.ResponseWriter, req *http.
 		MetricFormat:       "carbon2",
 		Client:             "otelcol",
 		MaxRequestBodySize: 20_971_520,
+		GraphiteTemplate:   DefaultGraphiteTemplate,
```

Comment:
Why is this needed here? Maybe we should use the default config via `createDefaultConfig` and then change it here?

Reply:
I agree it would be more readable to use the default config and override anything needed. I'll refactor that.

Reply:
Done @pmalek-sumo please take a look